include/crocoddyl/core/integrator/rk4.hxx

```diff
@@ -11,44 +11,62 @@
 #include "crocoddyl/core/utils/exception.hpp"
 #include "crocoddyl/core/integrator/rk4.hpp"
 
+using namespace std;
```

Comment:
Please do not include this line as it might introduce bugs. 
Instead,  use the `std` namespace in the respective code.